backend/src/main/java/capstone/storage/backend/storagebin/StorageBinController.java

```diff
@@ -0,0 +1,19 @@
+package capstone.storage.backend.storagebin;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/storage/")
```

Comment:
Bitte URLs im Plural, da bei get ein array zurückgeliefert wird.
```suggestion
@RequestMapping("api/storagebins")
```